code/datums/ai/dog/dog_controller.dm

```diff
@@ -75,6 +75,9 @@
 	else if(DT_PROB(10, delta_time))
 		living_pawn.manual_emote(pick("dances around.", "chases [living_pawn.p_their()] tail!</span>"))
 		living_pawn.AddComponent(/datum/component/spinny)
+		for (var/mob/i in oviewers(living_pawn))
+			if (ishuman(i) && i.client) //if viewer is human and a player. maybe check if isn't SSD? also, do non-human players have moods?
```

Comment:
check for mind not client

Reply:
Oh, okay. I'm new to SS13 code, so I didn't know about mind. Does the new commit do it right?

Reply:
Yes